handlers/routes.go

```diff
@@ -61,6 +61,9 @@ func (s *Server) routes() {
 	views.HandleFunc("/login", s.authGet()).Methods(http.MethodGet)
 	views.PathPrefix("/-{id}").HandlerFunc(s.entryGet()).Methods(http.MethodGet)
 	views.PathPrefix("/-{id}/{filename}").HandlerFunc(s.entryGet()).Methods(http.MethodGet)
+	// Allow POST to same path to submit passphrase form
+	views.PathPrefix("/-{id}").HandlerFunc(s.entryGet()).Methods(http.MethodPost)
```

Comment:
We need a separate handler function to handle POST requests. `entryGet` is strictly for `GET` requests.

We should also unit test new route handlers.